src/EventStore.Core/Services/Transport/Http/Controllers/AdminController.cs

```diff
@@ -64,6 +64,11 @@ public class AdminController : CommunicationController {
 				SupportedCodecs, ReadStreamOperationForScavengeStream);
 			Register(service, "/streams/$scavenges?embed={embed}", HttpMethod.Get, GetStreamEventsBackwardScavenges, Codec.NoCodecs,
 				SupportedCodecs, ReadStreamOperationForScavengeStream);
+			
+			// this exists only to specify the permissions required for the /admin/logs & /admin/logs/{file} endpoint
+			// Kestrel handles the request itself
+			RegisterAuthOnly(service, "/admin/logs", HttpMethod.Get, Codec.NoCodecs, new ICodec[]{Codec.Json}, Operations.Node.Shutdown);
```

Comment:
For a proper implementation we'd make an operation for getting the logs and probably a separate one for listing them

Reply:
i mean as opposed to using `Operations.Node.Shutdown`